src/Covenant.Tests/PythonPoetryTests.cs

```diff
@@ -0,0 +1,22 @@
+using Covenant.Analysis.Poetry;
+using Shouldly;
+using Spectre.IO;
+
+namespace Covenant.Tests;
+
+public class PythonPoetryTests
```

Comment:
Is this finished?

This seems to have a very lenient definition of `Correctly`. Shouldn't these tests inspect the results?

Also, there don't appear to be any test at all for `PoetryAnalyzer`. Isn't that type the heart of this PR? (I'm currently uncertain on what anlaysis `PoetryAnalyzer` is actually meant to do, and having some tests would help clarify that.)